data/basal_test.go

```diff
@@ -0,0 +1,177 @@
+package data
+
+import (
+	"github.com/tidepool-org/platform/validate"
+
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/ginkgo"
+	. "github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/onsi/gomega"
+)
+
+var _ = Describe("Basal", func() {
+
+	const (
+		userid   = "b676436f60"
```

Comment:
Should standardize on ID being all capitalized.  I think golint will pick it up if you tried userId and want userID.  So, let's be consistent.